core/commands/check_relevance_command.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require_relative 'base_command'
+
+# This class checks relevance network config
+class CheckRelevanceCommand < BaseCommand
```

Comment:
Class must provide an extended documentation on how to use this command. In order to do that:
1. Add a method that will print the broad description of the command. See `ConfigureCommand#show_help` for example.
2. In the execute step check that `--help` flag was submitted to the command. If it was, then show help and stop the execution.